### PR TITLE
Missing icons from Font Mfizz added

### LINKED ISSUE #683: Feature Request: missing icons from Font Mfizz
### Suggested Feature

Here is an abandoned project **Font Mfizz**

> Font Mfizz provides scalable vector icons representing programming languages, operating systems, software engineering, and technology. It was designed designed for technology and software geeks. It can be customized — size, color, drop shadow, and anything that can be done with the power of CSS.

The project is not updated since 2017 but has a good bunch of icons but it's still a good list of ideas for devicon.

- Source: https://github.com/fizzed/font-mfizz
- Demo: http://fizzed.com/oss/font-mfizz

### Why we should have this feature

Completing the missing icons from Font Mfizz.

### PR DESCRIPTION
**Pull request by soham4abc**
Fixes #683

### CODE REVIEW

**Comment by theblobscp** [comment]
Looks fine, but isn't blackberry a company?

**Comment by soham4abc** [comment]
> Looks fine, but isn't blackberry a company?

I had the same question in my mind... I think we do not need it. Am I right?

**Comment by theblobscp** [comment]
Yeah probably not

**Comment by soham4abc** [comment]
> Yeah probably not

removing it

**Comment by theblobscp** [comment]
Well other than that the pr is good

**Review by theblobscp — APPROVED** [review]
I said I approved before, just using the github feature to do it

**Comment by soham4abc** [comment]
> I said I approved before, just using the github feature to do it

Thank you!

**Comment by soham4abc** [comment]
@Panquesito7  Please do review!

**Comment by theblobscp** [comment]
oof

File: devicon.json [hunk 1]
```diff
@@ -362,7 +411,7 @@
             "alias": "plain-wordmark"
           }
         ]
-    },
+    },
```
Comment (Panquesito7) [inline devicon.json:414]:
```suggestion
    },
```

File: devicon.json [hunk 2]
```diff
@@ -1,4 +1,53 @@
...
+        "versions": {
+            "svg": [
+                "original"                
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#000000",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:17]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```

File: devicon.json [hunk 3]
```diff
@@ -1,4 +1,53 @@
...
+        "tags": [
+            "3d",
+            "design"
+        ],
+        "versions": {
+            "svg": [
+                "original"                
+            ],
+            "font": [
+                "plain"
```
Comment (Panquesito7) [inline devicon.json:13]:
```suggestion
                "original"
```

File: devicon.json [hunk 4]
```diff
@@ -1,4 +1,53 @@
...
+        "name": "alpinelinux",
+        "tags": [
+            "linux"           
+        ],
+        "versions": {
+            "svg": [
+                "original"                
+            ],
+            "font": [
+                "plain"
```
Comment (Panquesito7) [inline devicon.json:29]:
```suggestion
                "original"
```

File: devicon.json [hunk 5]
```diff
@@ -1,4 +1,53 @@
...
+            ]
+        },
+        "color": "#181aab",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:33]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```

File: devicon.json [hunk 6]
```diff
@@ -1,4 +1,53 @@
...
+        "name": "azure",
+        "tags": [
+            "linux"           
+        ],
+        "versions": {
+            "svg": [
+                "original"                
+            ],
+            "font": [
+                "plain"
```
Comment (Panquesito7) [inline devicon.json:45]:
```suggestion
                "original"
```

File: devicon.json [hunk 7]
```diff
@@ -1,4 +1,53 @@
...
+            ]
+        },
+        "color": "#666666",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:49]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```

File: devicon.json [hunk 8]
```diff
@@ -1016,6 +1065,24 @@
...
+        "tags": [
+            "framework"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+                
+            ],
+            "font": [
+                "plain"
```
Comment (Panquesito7) [inline devicon.json:1079]:
```suggestion
                "original"
```

File: devicon.json [hunk 9]
```diff
@@ -1016,6 +1065,24 @@
...
+                "original"
+                
+            ],
+            "font": [
+                "plain"            
+            ]
+            
+        },
+        "color": "#000000",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:1084]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by theblobscp** [comment]
No idea what you did but looks cool

**Comment by soham4abc** [comment]
@Panquesito7  All the requested changes have been merged. If anything else is required in this PR pls do let me know! Thanks for the support!

**Comment by theblobscp** [comment]
Thats a lot of changes

**Comment by Thomas-Boi** [comment]
Hi @soham4abc,

Thank you very much for your work! Overall, the icons look great. However, in order for our `peek-bot` to work, I'll need you to make each icon into a separate PR. So `alpinelinux` should be a new PR, `azure` should be another one, etc.  Please refer to step 4 and 5 [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview). 

If you are not sure how to achieve this with git, there's a few ways to do this:
- Checkout a new branch from `develop` and cherry-pick commits specific to an icon. Ex: create a `soham4abc:alpinelinux` branch, cherry pick the `alpinelinux`-related commits from this branch then push.
- Mixed reset everything so this PR branch is the same as `develop`. Then, create a commit for each icon added and checkout a new branch after the commit. Make sure the `devicon.json` only contains the changes for that specific icon. 
- Same as step 1 but rather than cherry-pick, copy and paste the files.

I'll close this PR for now. I hope to see your new icons soon 😃

**Comment by soham4abc** [comment]
> Hi @soham4abc,
> 
> Thank you very much for your work! Overall, the icons look great. However, in order for our `peek-bot` to work, I'll need you to make each icon into a separate PR. So `alpinelinux` should be a new PR, `azure` should be another one, etc. Please refer to step 4 and 5 [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview).
> 
> If you are not sure how to achieve this with git, there's a few ways to do this:
> 
> * Checkout a new branch from `develop` and cherry-pick commits specific to an icon. Ex: create a `soham4abc:alpinelinux` branch, cherry pick the `alpinelinux`-related commits from this branch then push.
> * Mixed reset everything so this PR branch is the same as `develop`. Then, create a commit for each icon added and checkout a new branch after the commit. Make sure the `devicon.json` only contains the changes for that specific icon.
> * Same as step 1 but rather than cherry-pick, copy and paste the files.
> 
> I'll close this PR for now. I hope to see your new icons soon 😃

Done the same... will give more PRs for each icon respectively. For 3dPrints #740 has been already opened! Please do have a look!

**Pull request closed without merging**